src/OpenTelemetry.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -23,7 +23,13 @@ private Settings(IConfigurationSource source)
                 throw new ArgumentNullException(nameof(source));
             }
 
-            Exporter = source.GetString(ConfigurationKeys.Exporter);
+            Exporter = source.GetString(ConfigurationKeys.Exporter) ??
+#if NETFRAMEWORK
+                       "zipkin";
```

Comment:
alternatively, for.NET Framework, we could still use the OTLP exporter, but set HTTP protocol